source/console/minio-console.rst

```diff
@@ -131,316 +131,413 @@ routing rules may require setting a static MinIO Console port. For example,
-   :alt: MinIO Console Healing
-   :align: center
+      - Use the :guilabel:`Filter` to apply text filters to the log results
+
+      Select the :guilabel:`Start Logs` button to begin collecting logs using the selected filters and settings.
+
+   .. tab-item:: Audit
+
+      The Audit Log section provides an interface for viewing :ref:`audit logs <minio-logging>` collected by a configured PostgreSQL service.
+
+      The Audit Logging feature is configured and enabled automatically for MinIO deployments created using the :docs-k8s:`MinIO Kubernetes Operator <>`.
+
+   .. tab-item:: Trace
+
+      .. image:: /images/minio-console/console-trace.png
+         :width: 600px
+         :alt: MinIO Console Trace
+         :align: center
+
+      The :guilabel:`Trace` section provides HTTP trace functionality for a bucket or buckets on the deployment. 
+      This section provides similar functionality to :mc:`mc admin trace`.
+
+      You can modify the trace to show only specific trace calls.
+      The default is to show only :guilabel:`S3` related HTTP traces.
+      
+      Select :guilabel:`Filters` to open additional filters to apply to trace output, such as restricting the :guilabel:`Path` on which the trace applies to a specific bucket or bucket prefix.
+
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
+
+   .. tab-item:: Watch
+
+      .. image:: /images/minio-console/console-watch.png
+         :width: 600px
+         :alt: MinIO Console Watch
+         :align: center
+
+      The :guilabel:`Watch` section displays S3 events as they occur on the selected bucket. 
+      This section provides similar functionality to :mc:`mc watch`.
 
-The :guilabel:`Heal` section displays the healing status for a bucket. 
-MinIO automatically heals objects and drives when it detects problems, such
-as drive-level corruption or a replacement drive.
+      This tab or its contents may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
 
-MinIO does not recommend performing manual healing unless explicitly directed
-by support. 
+   .. tab-item:: Drives
 
-This tab or its contents may
-not be visible if the authenticated user does not have the 
-:ref:`required administrative permissions <minio-policy-mc-admin-actions>`
+      .. image:: /images/minio-console/console-health.png
+         :width: 600px
+         :alt: MinIO Console Healing
+         :align: center
 
-Tools: Diagnostics
-------------------
+      The :guilabel:`Heal` section displays the healing status for a bucket. 
+      MinIO automatically heals objects and drives when it detects problems, such as drive-level corruption or a replacement drive.
 
-.. image:: /images/minio-console/console-diagnostics.png
+      MinIO does not recommend performing manual healing unless explicitly directed
```

Comment:
Consider making this a caution or warning admonition.

Reply:
Done